src/XCCDF_POLICY/xccdf_policy_remediate.c

```diff
@@ -1508,10 +1508,11 @@ static int _generate_kickstart_post(struct kickstart_commands *cmds, const char
 	char *basename = oscap_basename(dup);
 	free(dup);
 	char *oscap_command = oscap_sprintf(
-		"oscap xccdf eval --remediate --profile '%s' /usr/share/xml/scap/ssg/content/%s\n",
+		"oscap xccdf eval --remediate --results-arf /root/openscap_data/arf.xml --report /root/openscap_data/report.html --profile '%s' /usr/share/xml/scap/ssg/content/%s\n",
```

Comment:
How about we just write down results in `/root/oscap_arf_results.xml` and `/root/oscap_report.html` and skip creating a directory?

Reply:
It could be fine, but the directory separates and concentrates multiple related files in a single directory.

Reply:
In IB we put them right in the `/root` by default. Also, `oscap_data` does not follow [XDG specs](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html).

What actually worries me is the requirement to create this directory at some point. As our Kickstart will become more complex we could run in race condition with it. The `/root` OTOH is always there.

Reply:
I would maybe recommend `/var/log/anaconda` instead (for IB as well if it creates files in `/root`) - storing files in `/root` might become problematic with bootc images and the overall move towards immutable systems where `/etc` and `/var` are the only dynamic places (aside from `*/tmp`).

I'm not sure if `/var/log/anaconda` is always created, or if `inst.nosave=logs` disables just storing `.log` files or the creation of the directory as well, so that would need to be checked.

Reply:
I have now tried to replace `/root` by `/var/log/anaconda` and it doesn't work, the installation doesn't run the `oscap xccdf remediate` and in the logs after installation it says `No such file or directory`.